alvr/server/cpp/platform/win32/VideoEncoderVCE.cpp

```diff
@@ -224,25 +252,17 @@ amf::AMFComponentPtr VideoEncoderVCE::MakeEncoder(
 		//No noticable performance difference and should improve subjective quality by allocating more bits to smooth areas
 		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_HEVC_ENABLE_VBAQ, true);
 
-		//Fixes rythmic pixelation. I-frames were overcompressed on default settings
-		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_HEVC_MAX_QP_I, 30);
-	}
-	amf::AMFCapsPtr caps;
-	amfEncoder->GetCaps(&caps);
-	amf::AMFIOCapsPtr iocaps;
-	caps->GetOutputCaps(&iocaps);
-	amf_int32 formats = iocaps->GetNumOfFormats();
-	Debug("Supported formats num: %d\n", formats);
-	for (int i = 0; i < formats; i++) {
-		amf::AMF_SURFACE_FORMAT format;
-		bool isNative;
-		iocaps->GetFormatAt(i, &format, &isNative);
-		Debug("Accepts format: %d, native: %d", format, isNative);
+		//Fixes rythmic pixelation.
+		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_MAX_QP, 30);
```

Comment:
That's VCE header, not HEVC. For MAX_QP, there're `AMF_VIDEO_ENCODER_HEVC_MAX_QP_I` and `AMF_VIDEO_ENCODER_HEVC_MAX_QP_P`